### ISSUE TITLE
Office 20% samples only achieved 0.86 accuracy

### ISSUE THREAD

**Issue by Jensen-Su** [thuml/MTlearn#1]
As a quick reproduction, I simply added the following class to file `model_multi_task.py`:
```python
class AlexnetNoFc(nn.Module):
    def __init__(self):
        super(AlexnetNoFc, self).__init__()
        alexnet = models.alexnet(pretrained=True)
        self.features = alexnet.features
        self.classifier = nn.Sequential()
        for i in xrange(6):
            self.classifier.add_module("classifier" + str(i), alexnet.classifier[i])
        self.extract_feature_layers = nn.Sequential(self.features, self.classifier)
        self.in_features = alexnet.classifier[6].in_features

    def forward(self, x):
        x = self.features(x)
        x = x.view(x.size(0), -1)
        x = self.classifier(x)
        return x

    def output_num(self):
        return self.in_features
```
Then I changed the model in `train_multi_task.py` from `vgg16no_fc` to `alexnetno_fc` and uncommented the commented statements in `model_multi_task.py` to use the other two covariance. Also, I changed "lr" in code
```
parameter_dict = [{"params":self.shared_layers.module.parameters(), "lr":0}]
```
to 1 to enable finetuning.

Then I trained the model using `train_20.txt` and `test_20.txt` of `Office` datasets. But I only got 0.86 accuracy, worse then single-task training and far from 0.94 reported in the paper. Did I missing anything?
